kmir/src/kmir/rust/cargo.py

```diff
@@ -80,3 +92,13 @@ def default_target(self) -> str:
             )
 
         return bin_targets[0]
+
+
+def cargo_get_smir_json(rs_file: Path) -> dict[str, Any]:
+    smir_json_result = SMIR_JSON_DIR / rs_file.with_suffix('.smir.json').name
+    run_process_2(['cargo', 'run', '--', '-Zno-codegen', str(rs_file)], cwd=SMIR_JSON_DIR)
+    json_smir = json.loads(smir_json_result.read_text())
```

Comment:
Remark: This does not actually have to use `cargo`, it is a single crate because it starts from a single Rust file.
We don't have linking for multi-crate programs atm, but let's still use cargo here (recompilation might become a bit tricky, though)